### DISCUSSION TITLE
Crazyflie tutorial Regards

### DISCUSSION THREAD

**Discussion by jeetu-8910** [bitcraze/discussions#849] [Q&A]
I am Ashutosh Dewangan , I want to convey a little problem which me and my friends (Priyanshu Sidar , Jitendra Kumar Dewangan) are facing while doing a project using crazyflie. My friends and I are looking for tutorials for the use of the **_Cflib library_** which is used in our project to make the project successful. The motto of our project is to make our crazyflie revolve in a certain desired path, for which the tutorial is required. Please look into the matter.  Thank You Yours Sincerely Ashutosh Dewangan Priyanshu Sidar Jitendra Kumar Dewangan

**Comment by knmcguire** [comment]
Hi! Have you tried the step-by-step guides in the cflib documentation yet? https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/. Especially the motion commander one will be intersesting for you.

Also take a look at the autonomy examples of the [cflib repository](https://github.com/bitcraze/crazyflie-lib-python)

**Comment by jeetu-8910** [comment]
Mam ,actually we want to work on cflib from scratch , so that we can
perform any kind of experient not only the one that you have provided . So
kindly inform if any documentation to learn cflib is present or not .

**Reply by knmcguire** [reply]
To learn the cflib, the step-by-step guides are made exactly for that, starting with the logging and parameter framework: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/

This is with more generic explanation of the API: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/python_api/

But there is also automated documentation made as well: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/api/cflib/

This is all we have in terms of written documentation.

**Comment by jeetu-8910** [comment]
Thank you for the help, but madam it's a humble request can u please share
us the live/video demonstration of swarm interface as we are facing some
difficulties to proceed in our project. Kindly look into it Madam.

**Reply by knmcguire** [reply]
Sorry, there is no live/video demostration of the swarm interface. Please start a new discussion thread and try to explain your issue as much as you can, with error codes and which scripts are you using
